Regulator_SwitchedCapacitor.dcm

```diff
@@ -24,6 +24,12 @@ K Low-noise inverting charge pump with both positive and negative LDO's
 F http://www.ti.com/lit/ds/symlink/lm27762.pdf
 $ENDCMP
 #
+$CMP LM7705
+D Switched capacitor inverter, inverts 3V to 5.25V to regulated negative 0.23V, VSSOP-8
```

Comment:
Please let me know what you think of this description: `Negative bias generator, low-noise, 3-5.25V input, -0.232V fixed output, VSSOP-8`